### PR TITLE
Add ab2d-ops create-analyst-view.sql

### PR DESCRIPTION
**Pull request by sb-dkniesteadt**
**JIRA Tickets:**

[AB2D-4391](https://jira.cms.gov/browse/AB2D-4391) - Description
 
### What Does This PR Do?
Moves sql scripts from ab2d-ops to the main repo

### What Should Reviewers Watch For?
Any thing nneded to add in the sql script

### Security Implications

- [ ] This PR adds new software dependencies
- [ ] This PR modifies or invalidates our security controls
- [ ] This PR stores or transmits data that was not stored or transmitted before

- [ ] This PR requires additional review of its security implications for other reasons

### Submitter Checklist

- [x] This PR includes any required documentation changes, (`README`, changelog / release notes, website).
- [x] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments.
- [x] Code checked for PHI/PII exposure

### CODE REVIEW

File: common/src/main/resources/db/changelog/v2022/create-analyst-view.sql
Comment (nflearl) [inline common/src/main/resources/db/changelog/v2022/create-analyst-view.sql:142]: Check with Will but I don't think you want anything to do with the analyst in this checkin.

**Review by nflearl — CHANGES_REQUESTED** [review]
What about the bcp_view?

File: common/src/main/resources/db/changelog/v2022/create-analyst-view.sql
Comment (sb-dkniesteadt) [reply common/src/main/resources/db/changelog/v2022/create-analyst-view.sql:142]: @wnyffenegger Comment?

Comment (sb-wnyffenegger) [reply common/src/main/resources/db/changelog/v2022/create-analyst-view.sql:142]: Well I can't control permissions from ab2d-ops if those views don't already exist so I would include everything.

Comment (sb-wnyffenegger) [reply common/src/main/resources/db/changelog/v2022/create-analyst-view.sql:142]: We're assuming that the ab2d_analyst is already created in the environment and you're just populating the views it can use. It starts with zero view of anything so you're going to have to do the grants.

Comment (sb-wnyffenegger) [reply common/src/main/resources/db/changelog/v2022/create-analyst-view.sql:142]: @sb-dkniesteadt one thing though, get rid of the create role part. Assume that the role already exists in the environment.

Comment (sb-wnyffenegger) [reply common/src/main/resources/db/changelog/v2022/create-analyst-view.sql:142]: Tricky, I don't think so. Maybe that changes things. I don't want to deal with that mess. It's a chicken and egg problem because during the deployment of the database these views won't exist. It's only when a worker/api start up that it will.

**Review by sb-wnyffenegger — COMMENTED** [review]
Approved after merge in ab2d-ops

**Review by sb-ebukaanene — APPROVED** [review]

**Pull request merged**
